src/openforms/submissions/attachments.py

```diff
@@ -257,7 +259,7 @@ def attach_uploads_to_submission_step(submission_step: SubmissionStep) -> list:
             # NOTE there is a possible race-condition if user completes a submission before this resize task is done
             # see https://github.com/open-formulieren/open-forms/issues/507
             transaction.on_commit(
-                lambda: resize_submission_attachment.delay(attachment.id, resize_size)
+                resize_submission_attachment.si(attachment.id, resize_size).delay
```

Comment:
I thought we were using `si` when we want to have a chain of tasks. is this serving similarly our purpose?

Reply:
`si` is to make a "partial" signature of the task with some arguments (and is immutable, but not really relevant in our case. `s` is a way to make a signature but mutable).

It's similar to `functools.partial` in some way

Reply:
This was my initial thought, but then thinking about about `s`/`si` I was wondering why they existed if `partial` is doing the same thing? Is there something that `s`/`si` does that `partial` doesn't?

Reply:
`.si` and friends are specific to chains because by default the return value of a previous task in a chain is passed as an argument to the next task, which `.si` prevents from happening.